Makefile

```diff
@@ -66,6 +66,8 @@ MLRUN_PYTHON_VERSION_SUFFIX := $(if $(INCLUDE_PYTHON_VERSION_SUFFIX),$(MLRUN_ANA
 MLRUN_OLD_VERSION_ESCAPED = $(shell echo "$(MLRUN_OLD_VERSION)" | sed 's/\./\\\./g')
 MLRUN_BC_TESTS_OPENAPI_OUTPUT_PATH ?= $(shell pwd)
 
+CHECKED_IN_PYTHON_FILES := $(shell git ls-files | grep '\.py$$')
```

Comment:
This doesn't catch notebooks

Reply:
The current code doesn't run on the notebooks either.